frontend/tsconfig.json

```
@@ -41,7 +41,8 @@
       "@pages": ["pages/index.ts"],
       "@pages/*": ["pages/*"]
     },
-    "typeRoots": ["src/custom-types"]
+    "typeRoots": ["src/custom-types"],
+    "types": ["cypress", "@testing-library/cypress"]
```

```suggestion
    "types": ["cypress", "@testing-library/cypress"] # https://docs.cypress.io/guides/tooling/typescript-support#Configure-tsconfig-json
```
설정한 이유를 링크로 남기는건 어떨까요?